Diagnostics/HealthChecker/DataCollection/ExchangeInformation/Tests/Get-ExchangeConnectors.Tests.ps1

```diff
@@ -130,11 +130,13 @@ Describe "Testing Get-ExchangeConnectors.ps1" {
             ($results[5].CertificateDetails.CertificateLifetimeInfo).Count | Should -Be 2
             foreach ($key in ($results[5].CertificateDetails.CertificateLifetimeInfo).keys) {
                 if ($key -eq "E267D459A0FB53D0EF225C11FAC062D522648C09") {
-                    ($results[5].CertificateDetails.CertificateLifetimeInfo)[$key] | Should -Be 1678
+                    $testDays = ([System.Convert]::ToDateTime($multipleMatchingExchangeCertificates[3].CertificateObject.NotAfter, [System.Globalization.DateTimeFormatInfo]::InvariantInfo) - (Get-Date)).Days
```

Comment:
While this does work, not sure if this is the best way to go about the test.

Reply:
Would this be a test case that we need to have? We could hard code the UTC time here, similar to what we had before, that might be better.

Reply:
I've switched to using the `BaseUtcOffset` (https://docs.microsoft.com/en-us/dotnet/standard/datetime/converting-between-datetime-and-offset) to make sure that the `DateTime` that is being used in the unit test is the same on all computers (independent of the local configured time zone). I've added a `ConvertTimeToUtcHelper` function to perform the operations.